cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -343,13 +343,13 @@ def test_invalid_paging(self, page_number, page_size):
     def test_get_user_partitions_and_groups(self):
         self.course.user_partitions = [
             UserPartition(
-                id=0,
+                id=100,
```

Comment:
Note to reviewers: Changing IDs so they more accurately represent what the code does (and avoid non-unique IDs).